### DISCUSSION TITLE
Exploring the possible usage of this library for connecting with a Bluetooth supporting microcontroller. ESP32 in my case.

### DISCUSSION THREAD

**Discussion by Amritanshu-Shandilya** [hbldh/bleak#1393] [General]
Hi. I am trying to make a smartwatch using esp32. [This is the code repository ](https://github.com/Amritanshu-Shandilya/Mini_watch)
Also to complicate things and to open an entirely new domain of functionalities I planned to write an companion app for the smart watch in python(, scary right? It is). As you might have guessed it why I wanted to use _bleak_ . I am using beeware for developing the companion app and it is still in its infancy, you could have a look of it if you want :
[APP](https://github.com/Amritanshu-Shandilya/mini-watch-companion)

I have some questions like :

- have you used / or came across someone who has used bleak for connecting with a microcontroller ?
- does bleak works for android? like is it capable of accessing Bluetooth hardware of smartphones?
also many more. I hope you would reply to this as soon as possible. It would be very helpful for my project if you could connect with me here as well as in twitter. My ID is senku_of_India 

Also this isnt a commercial project. Neither it is an assignment for my college. I am just building this stuff because I want to learn stuff like this and one day develop something worth. Have a nice day. Thank you.

**Comment by dlech** [comment]
Bleak for Android is not in great shape. We currently have an implementation using Python4Android (Kivy). We don't have anything for Beeware currently. Contributions are welcome.
